climpred/bootstrap.py

```diff
@@ -441,6 +443,7 @@ def bootstrap_compute(
             compute(
                 uninit_hind,
                 verif,
+                alignment=alignment,
```

Comment:
Here I think the hist alignment could be implemented. But when the tests pass and function still works it’s ok for me. It will change in the uninit reference compute refactoring

Reply:
Yeah it's being handled here since it's still going through the main compute functions. When we refactor we'll implement the new method.

Reply:
so currently hist is not aligned. the refactoring PR then should handle this. Maybe hist can be an optional argument in the alignment function and then a union with hist/obs/hind should probably do it.
```
hind minmax 1954-01-01 00:00:00 2017-01-01 00:00:00
hist minmax 1955-01-01 00:00:00 2015-01-01 00:00:00
veri minmax 1948-01-01 00:00:00 2017-01-01 00:00:00

 same_verif
INFO:root:`compute_hindcast` for metric pearson_r and comparison e2o at 2020-03-04 22:10:05.002853
++++++++++++++++++++++++++++++++++++++++++++++++
INFO:root:lead=01 | dim=time | inits=1956-01-01 00:00:00-2016-01-01 00:00:00 | verif=1957-01-01 00:00:00-2017-01-01 00:00:00
INFO:root:lead=02 | dim=time | inits=1955-01-01 00:00:00-2015-01-01 00:00:00 | verif=1957-01-01 00:00:00-2017-01-01 00:00:00
INFO:root:lead=03 | dim=time | inits=1954-01-01 00:00:00-2014-01-01 00:00:00 | verif=1957-01-01 00:00:00-2017-01-01 00:00:00
INFO:root:`compute_hindcast` for metric pearson_r and comparison e2o at 2020-03-04 22:10:05.412592
++++++++++++++++++++++++++++++++++++++++++++++++
INFO:root:lead=01 | dim=time | inits=1957-01-01 00:00:00-2012-01-01 00:00:00 | verif=1958-01-01 00:00:00-2013-01-01 00:00:00
INFO:root:lead=02 | dim=time | inits=1956-01-01 00:00:00-2011-01-01 00:00:00 | verif=1958-01-01 00:00:00-2013-01-01 00:00:00
INFO:root:lead=03 | dim=time | inits=1955-01-01 00:00:00-2010-01-01 00:00:00 | verif=1958-01-01 00:00:00-2013-01-01 00:00:00
INFO:root:`compute_hindcast` for metric pearson_r and comparison e2o at 2020-03-04 22:10:05.798249
++++++++++++++++++++++++++++++++++++++++++++++++
INFO:root:lead=01 | dim=time | inits=1956-01-01 00:00:00-2016-01-01 00:00:00 | verif=1957-01-01 00:00:00-2017-01-01 00:00:00
INFO:root:lead=02 | dim=time | inits=1955-01-01 00:00:00-2015-01-01 00:00:00 | verif=1957-01-01 00:00:00-2017-01-01 00:00:00
INFO:root:lead=03 | dim=time | inits=1954-01-01 00:00:00-2014-01-01 00:00:00 | verif=1957-01-01 00:00:00-2017-01-01 00:00:00

Reply:
I see what you mean. Yeah I put a kludge into `compute_uninitialized` right now to find that union.